test/gerkin/tests/features/db_delete.feature

```diff
@@ -0,0 +1,14 @@
+Feature: API endpoint that allows user to delete database schema.
+  Request endpoint: DELETE /database/{id}/
+
+  Background:
+    Given database with the valid schema has been set up
+
+  Scenario: Successfully delete database schema
+    When I make a DELETE request with a valid payload
```

Comment:
Same case as for `data` DELETE statement, it doesn't have payload, only ID in URL path. Additionally `Information-Mediator-Client` is required as a `Header`. This header is a must have for all /database/ requests.